src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -93,11 +95,16 @@ public UserDefault getUserDefaultSettings(final String userId) {
             final HttpEntity requestEntity = new HttpEntity(securityUtils.authorizationHeaders());
             final Map<String, String> queryParams = new HashMap<>();
             queryParams.put("uid", userId);
+            final DefaultUriBuilderFactory builderFactory = new DefaultUriBuilderFactory();
+            // The EncodingMode.VALUES_ONLY mode ensures the query string variables are encoded
+            builderFactory.setEncodingMode(EncodingMode.VALUES_ONLY);
+            restTemplate.setUriTemplateHandler(builderFactory);
```

Comment:
May be the right place to set any properties on restTemplate is during the bean creation in RestTemplateConfiguration as restTemplate is an autowired bean here.  

Alternatively, to encode the user id, you could use the static method ApplicationParams.encodeBase64 - safer and isolated change. Setting properties on restTemplate may need regression testing on all endpoints where restTemplate is used.

Reply:
Ideally, the query parameters just need `UTF-8` encoding, so `Base64` might be a bit overkill. Setting the encoding mode on `restTemplate` here - wouldn't this affect only the instance local to this class? I think it's quite safe, given `EncodingMode.VALUES_ONLY` targets only URI variables and not the whole URI template.

Reply:
@Daniel-Lam I thought restTemplate is an autowired singleton bean and not a local variable, I may be wrong. There is another static method ApplicationParams.encode (just need to make it public), that encodes to UTF-8. Whatever you think is right, go for it 🙂

Reply:
@gbenadikar - I think you're correct, actually... it is an autowired singleton. There will be a future ticket to ensure _all_ requests using `RestTemplate` encode their URI variables, but for now I think I'll make `ApplicationParams.encode()` public and use that instead.